src/org/opensolaris/opengrok/web/PageConfig.java

```diff
@@ -752,6 +752,20 @@ public String getRequestedProjectsAsString() {
         return requestedProjects;
     }
     
+	public SortedSet<String> getAllRequestedProjects() {
```

Comment:
Also isn't better idea to move the check of `is this a search all?` to the `getRequestedProjects` and act upon there? It would make the change more transparent I think